src/components/Router.tsx

```diff
@@ -15,7 +16,11 @@ export const Router: React.FC = ({ children }) => (
   <Switch>
     <Route path={`${routeBasePath}/clusters/~new`} component={NewClusterPage} />
```

Comment:
Let's move this inside `SingleCluster` to reduce chance of 2nd cluster creation.